### PR TITLE
Enforce line length of 80 chars

### PR DESCRIPTION
**Pull request by jbischof**
Line length is currently set at 88 chars. Bring this back to 80 chars to agree with Keras and KerasNLP.

Note that this does not work for docstrings and comments. We have exempted them to allow for hyperlinks.

### CODE REVIEW

**Review by qlzh727 — APPROVED** [review]
Thanks for the fix.

File: shell/lint.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ then
   exit 1
 fi
 [ $# -eq 0 ] && echo "no issues with flake8"
-black --check $files
+black --line-length 80 --check $files
```
Comment (bhack) [inline shell/lint.sh:34]: I suggest to move this in the configuration file or we misaling IDE integrations.

**Review by LukeWood — APPROVED** [review]
fine by me as long as @haifeng-jin has no specific reason for 88.

we took 88 from keras-tuner, dont think it should be an issue as long as it is uniform.  LGTM

File: shell/lint.sh (hunk 1 above)
Comment (jbischof) [reply shell/lint.sh:34]: @bhack I like your suggestion but let's save this for a followup PR. We need to find a unified solution that works across all `keras-team` repos long term.

**Comment by jbischof** [comment]
\gcbrun

**Pull request merged**